ops/cli.py

```diff
@@ -1,351 +1,309 @@
+            raise RuntimeError(f"GKE cluster {config.cluster_name} not found")
+        else:
+            success(f"GKE cluster {config.cluster_name} exists")
+
+
+def infer_missing_arguments(config: WflInstanceConfig) -> None:
+    """Infer and store all arguments not required for validation steps."""
+    if not config.version:
+        with open("version") as version_file:
+            config.version = version_file.read().strip()
+    if not config.cloud_sql_name:
+        config.cloud_sql_name = \
+            json.loads(shell(f"gcloud --project {config.project} --format=json "
+                             "sql instances list "
+                             f"--filter='labels.app_name=wfl AND labels.instance_id={config.instance_id}'",
+                             quiet=True))[0]["name"]
+    if not config.cluster_name:
+        config.cluster_name = \
+            json.loads(shell(f"gcloud --project {config.project} --format=json "
+                             "sql instances list "
+                             f"--filter='labels.app_name=wfl AND labels.instance_id={config.instance_id}'",
+                             quiet=True))[0]["settings"]["userLabels"]["app_cluster"]
+    if not config.cluster_zone:
+        config.cluster_zone = json.loads(shell(f"gcloud --project {config.project} --format=json "
+                                               "container clusters list "
+                                               f"--filter='name={config.cluster_name}'",
+                                               quiet=True))[0]["zone"]
+    if not config.cluster_namespace:
+        config.cluster_namespace = f"{config.instance_id}-wfl"
+
+
+def print_config(config: WflInstanceConfig) -> None:
+    """Print the entire config--be careful about calling this after sensitive info has been stored."""
+    PrettyPrinter().pprint(config)
+
+
+def configure_kubectl(config: WflInstanceConfig) -> None:
+    """Configure kubectl to use the stored cluster/namespace."""
+    info(f"=>  Configuring Kubernetes for cluster {config.cluster_name}")
+    shell(f"gcloud --project {config.project} container clusters get-credentials "
+          f"{config.cluster_name} --zone {config.cluster_zone}")
+    ctx = f"gke_{config.project}_{config.cluster_zone}_{config.cluster_name}"
+    shell(f"kubectl config use-context {ctx}")
+    try:
+        namespaces = json.loads(shell("kubectl get namespace -o json", timeout=5))
+        if config.cluster_namespace not in [item["metadata"]["name"] for item in namespaces["items"]]:
+            info(f"    Available namespaces in {config.cluster_name}:", plain=True)
+            info(shell("kubectl get namespace", quiet=True), plain=True)
+            raise RuntimeError(f"Namespace {config.cluster_namespace} not found in {config.cluster_name}")
+        else:
+            shell(f"kubectl config set-context --current --namespace={config.cluster_namespace}")
+            success("Kubernetes configured")
```

Comment:
This used to be a validation step, but no longer. Configuring kubectl now uses a reincarnation of some of the same logic to just print very helpful error text if the namespace doesn't exist.